src/test/java/net/snowflake/client/jdbc/ConnectionLatestIT.java

```diff
@@ -1168,6 +1169,7 @@ public void testReadOnly() throws Throwable {
   }
 
   @Test
+  @ConditionalIgnoreRule.ConditionalIgnore(condition = RunningNotOnAWS.class)
```

Comment:
for the conditional ignore let's add info why do we skip the tests on some environments

Reply:
@sfc-gh-dprzybysz Done